web-ui/src/main/resources/catalog/components/search/mdview/mdviewModule.js

```diff
@@ -104,6 +104,16 @@
       $scope.showStatusTopBarFor =
         gnGlobalSettings.gnCfg.mods.recordview.showStatusTopBarFor;
 
+      $scope.showStaticPageMenu =
```

Comment:
I'm not sure we need an extra setting for this. I mean if the user creates entries with the new menu option, why not not show them always instead of adding an extra UI option? What do you think @fxprunayre?